plugins/doc_fragments/docker.py

```diff
@@ -27,6 +28,9 @@ class ModuleDocFragment(object):
             - When verifying the authenticity of the Docker Host server, provide the expected name of the server.
             - If the value is not specified in the task, the value of environment variable C(DOCKER_TLS_HOSTNAME) will
               be used instead. If the environment variable is not set, the default value will be used.
+            - Due to an implementation error in this module, if you are connecting to a host other than C(localhost)
+              using TLS, you B(must) set this parameter (or C(DOCKER_TLS_HOSTNAME)) to the remote hostname.  This issue
+              will be corrected in a future version of this module.
```

Comment:
It is **not** an error, it is an unfortunate design choice. Also there is no guarantee that a docker daemon running on `localhost` has a certificate which includes `localhost`. And finally, this is not necessary a module, but the same doc fragment is also used in plugins.

Besides that, all necessary information is already given in the first part of the description.
```suggestion
            - If you obtain errors such as C(hostname 'localhost' doesn't match
'docker.local'), you probably have to set this option to the correct value.
```